java/fury-core/src/main/java/org/apache/fury/config/Config.java

```diff
@@ -133,6 +135,11 @@ public boolean deserializeNonexistentEnumValueAsNull() {
     return deserializeNonexistentEnumValueAsNull;
   }
 
+  /** deserialize and serialize enum by name. */
+  public boolean serializeEnumByName() {
```

Comment:
could we add this option to java guide options part in https://fury.apache.org/docs/guide/java_object_graph_guide/#furybuilder--options ,which correspond to  docs/guide/java_serialization_guide.md